File: Lab13-C#/Lab10/Fork.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Lab10
{
    [Serializable]
    class Fork:Statement
    {
        public Statement f;
	    public Fork(Statement s){
		    this.f=s;
	    }
	    public Statement getStatement()
	    {
		    return f;
	    }
	   public override string ToString() {
		    string result = "Fork( ";
		    result += f.ToString()+")";
		    return result;
	    }
    }
}

File: Lab13-Java/src/Model/Fork.java
package Model;

import java.io.Serializable;



public class Fork extends Statement implements Serializable
{
	private Statement st;
	
	public Fork(Statement s)
	{
		st=s;
	}
	
	public Statement getStatement()
	{
		return st;
	}
	
	public String ToString()
	{
		return "fork ( "+st.toString()+" )";
	}
}
